share/spice/dogo_news/footer.handlebars

```diff
@@ -0,0 +1,8 @@
+<div class="dogo_news_tile_footer">
```

Comment:
We actually wrap the footer in a div with class `tile__foot`. We can probably remove this wrapping div and adjust the CSS above